cmd/flightctl-server/main.go

```diff
@@ -15,9 +15,11 @@ const (
 	caCertValidityDays          = 365 * 10
 	serverCertValidityDays      = 365 * 1
 	clientBootStrapValidityDays = 365 * 1
+	apiAdminValidityDays        = 365 * 1
 	signerCertName              = "ca"
 	serverCertName              = "server"
 	clientBootstrapCertName     = "client-enrollment"
+	apiAdminCertificate         = "api-admin"
```

Comment:
"api-admin" sounds strange. As if it was the API being administrated or the admin was for the API only. Maybe just "admin" or "flightctl-admin"?